From c583e535a0196eaf762cb9942264d19673fe7337 Mon Sep 17 00:00:00 2001
From: "etham.li" <etham.li@unisoc.com>
Date: Fri, 18 Nov 2022 13:32:51 +0800
Subject: [PATCH] Bug #2081854 - [rsyslog] Modify test/events/scripts
 AC_ARG_ENABLE in configure.ac

[root cause  ]:Modify test/events/scripts AC_ARG_ENABLE in configure.ac
[changes     ]:Modify test/events/scripts AC_ARG_ENABLE in configure.ac
[side effects]:no
[self test   ]:build pass
[download normally]:yes
[power on/off normally]:yes
[do common repository/branch inspection]:no
[is there dependence]:no
[confirm dependent commit]:no
[board]:glp2.0
[test case]:build pass
[reviewers]:etham.li

commit_template_version:v1
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6f617c5..684faa7 100755
--- a/configure.ac
+++ b/configure.ac
@@ -587,19 +587,19 @@ AC_ARG_ENABLE(test,
           no) enable_test="no" ;;
            *) AC_MSG_ERROR(bad value ${enableval} for --enable-test) ;;
          esac],
-        [enable_test="yes"]
+        [enable_test="no"]
 )
 AM_CONDITIONAL(ENABLE_IMTEST, test x$enable_test = xyes)
 
 # events
-AC_ARG_ENABLE(test,
+AC_ARG_ENABLE(events,
         [AS_HELP_STRING([--enable-events],[Integrated events functionality @<:@default=yes@:>@])],
         [case "${enableval}" in
          yes) enable_events="yes" ;;
           no) enable_events="no" ;;
            *) AC_MSG_ERROR(bad value ${enableval} for --enable-events) ;;
          esac],
-        [enable_events="yes"]
+        [enable_events="no"]
 )
 AM_CONDITIONAL(ENABLE_IMEVENTS, test x$enable_events = xyes)
 
@@ -611,7 +611,7 @@ AC_ARG_ENABLE(script,
           no) enable_script="no" ;;
            *) AC_MSG_ERROR(bad value ${enableval} for --enable-script) ;;
          esac],
-        [enable_script="yes"]
+        [enable_script="no"]
 )
 AM_CONDITIONAL(ENABLE_IMSCRIPT, test x$enable_script = xyes)
 
-- 
2.17.1

